web/themes/custom/fmcg_material/sass/styles.scss

```diff
@@ -964,97 +914,18 @@ span.upsell-price .tax {
   min-height: 0;
 }
 
-.front-page-luova-block {
-  background-image: url("../assets/LuovaBanner868x250.png");
-  background-repeat: no-repeat;
-  background-size: cover;
-  margin-top: -15px;
-  padding-bottom: 24px;
-  border-bottom: 1px solid #FFFFFF;
-  position: relative;
-
-  h2 {
-    color: #FFFFFF;
-    font-weight: bolder;
-    line-height: 1.25em;
-    font-size: 2.5em;
-  }
-
-  p {
-    color: #FFFFFF;
-    font-size: 20px;
-    padding-right: 150px;
-  }
-
-  a:hover {
-    color: #49BCFB;
-  }
-
-  .btn {
-    margin-top: 15px;
-    margin-left: 0px;
-    background-color: #00B0FF;
-    color: #FFFFFF;
-    font-size: 1.2em;
-    padding: 4px 10px;
-
-    &:hover {
-      border: 2px solid #00B0FF;
-      background-color: #FFFFFF;
-      color: #00B0FF;
-    }
-  }
-
-  .luova-education-group {
-    color: #FF9049;
-  }
-  .victory-christian-international-school {
-    color: #FF9049;
-  }
-  .luova-banner-logo {
-    max-width: 220px;
-  }
-  .luova-logo-wrapper {
-    text-align: center;
-  }
-
-  @media (min-width: $md) {
-    background-image: url("../assets/LuovaBanner753x347.png");
-
-    .luova-banner-logo {
-      display: block;
-      width: 100%;
-    }
-  }
-  @media (min-width: $lg) {
-    background-image: url("../assets/LuovaBanner1905x433.png");
-
-    h2 {
-      font-size: 3.5em;
-    }
-  }
-}
-
 .section-wrapper {
   padding-bottom: 30px;
   padding-top: 0;
 }
 .section-wrapper .section {
-  max-width: 1200px;
+  max-width: 800px;
```

Comment:
This is not wide generally? Are we *sure* we want to limit every section to 800px?

Reply:
you might want to consider leaving .section and .section-wrapper alone and putting a mac-width on the <form> tag instead with margin-left and margin-right as auto.